src/stack/Stack.java

```diff
@@ -44,13 +55,21 @@ public void clear() {
 	 * @return
 	 */
 	public boolean isEmpty() {
-		return false;
-	}
 
-	/**
-	 * Increase the stack capacity if there is not enough space to add additional items
-	 */
-	private void ensureCapacity() {
+		if (tos > 0){
+			System.out.println(data[tos]);
+		}
+		else{
+			return true;
+		}
 
-	}
-}
\ No newline at end of file
+
+
+
+		/**
+		 * Increase the stack capacity if there is not enough space to add additional items
+		 */
+		private void ensureCapacity() {
+
+		}
+	}
```

Comment:
there is no implementation of the ensureCapacity() method and general stack testing.